index.js

```diff
@@ -142,6 +146,25 @@ function storeOption(longOption, optionValue, options, values) {
   }
 }
 
+/**
+ * Store the default option value in `values`.
+ *
+ * @param {string} longOption - long option name e.g. 'foo'
+ * @param {string
+ *         | boolean
+ *         | string[]
+ *         | boolean[]} optionValue - default value from option config
+ * @param {object} options - option configs, from parseArgs({ options })
+ * @param {object} values - option values returned in `values` by parseArgs
+ */
+function storeDefaultOption(longOption, optionValue, options, values) {
```

Comment:
No longer using options argument

Reply:
I wanted to keep the same interface as `storeOption`, but yes, it is unnecessary